airbyte-integrations/connectors/source-rss/source_rss/source.py

```diff
@@ -0,0 +1,155 @@
+#
+# Copyright (c) 2022 Airbyte, Inc., all rights reserved.
+#
+
+
+from abc import ABC
+from datetime import datetime
+from time import mktime
+from typing import Any, Iterable, List, Mapping, MutableMapping, Optional, Tuple
+
+import feedparser
+import pytz
+import requests
+from airbyte_cdk.sources import AbstractSource
+from airbyte_cdk.sources.streams import Stream
+from airbyte_cdk.sources.streams.http import HttpStream
+from dateutil.parser import parse
+
+item_keys = [
+    "title",
+    "link",
+    "description",
+    "author",
+    "category",
+    "comments",
+    "enclosure",
+    "guid",
+]
+
+
+def convert_item_to_mapping(item) -> Mapping:
+    mapping = {}
+
+    for item_key in item_keys:
+        try:
+            mapping[item_key] = item[item_key]
+        except (AttributeError, KeyError):
+            pass
+
+    try:
+        # get datetime in UTC
+        dt = datetime.fromtimestamp(mktime(item.published_parsed))
```

Comment:
@jrhizor some reason this is causing issues in Github actions but not on my local testing. Do you think we have issues with the mock across tz's?

Reply:
It looks like the problem was using `mktime` instead of `timegm`. I also added to the test case to make sure it tests with different Python timezone settings, which would have caught this error.